app/src/main/java/com/wireguard/android/activity/MainActivity.java

```diff
@@ -33,16 +31,10 @@
         implements FragmentManager.OnBackStackChangedListener {
     @Nullable private ActionBar actionBar;
     private boolean isTwoPaneLayout;
-    @Nullable private TunnelListFragment listFragment;
 
     @Override
     public void onBackPressed() {
         final int backStackEntries = getSupportFragmentManager().getBackStackEntryCount();
-        // If the action menu is visible and expanded, collapse it instead of navigating back.
-        if (isTwoPaneLayout || backStackEntries == 0) {
```

Comment:
Does this break two-pane mode on tablets?

Reply:
I'm confident it doesn't but I'll re-check with an emulator and attach a screenshot here.

Reply:
The answer to this question probably lies in the answer to "why was this code there before?"

Reply:
It was there because the action menu was. "if we're on a tablet or in the list fragment, and the action menu is collapsible, don't process onBackPressed"

Reply:
The "new fab" menu situation means this doesn't matter now? Okay then.